src/Wpf.Ui/Controls/SplitButton/SplitButton.xaml

```diff
@@ -134,7 +136,7 @@
                                     Focusable="False"
                                     Foreground="{TemplateBinding Foreground}"
                                     IsChecked="{TemplateBinding IsDropDownOpen}"
-                                    Style="{StaticResource DefaultSplitButtonToggleButtonStyle}">
+                                    Style="{DynamicResource DefaultSplitButtonToggleButtonStyle}">
```

Comment:
So we can override the style of the toggle button without having to redeclare the entire template.